src/kudu/thrift/sasl_client_transport.h

```diff
@@ -88,7 +88,7 @@ class SaslClientTransport
 
   ~SaslClientTransport() override = default;
 
-  bool isOpen() override;
+  bool isOpen() const override;
```

Comment:
fbcode's version of apache-thrift defined the parent method to be const so we have to make this const here too (it is const after all).

Child methods can be more restrictive but not less, so this still works here on a older version of apache-thrift.